Shared/Models/CampusEatery.swift

```diff
@@ -310,6 +310,73 @@ extension CampusEatery {
         return false
     }
 
+    /// A "major" meal of the day: breakfast, lunch, or dinner
+    enum MajorMeal: Int, CaseIterable, CustomStringConvertible, Comparable {
+
+        case breakfast
+        case lunch
+        case dinner
+
+        static func < (lhs: MajorMeal, rhs: MajorMeal) -> Bool {
+            lhs.rawValue < rhs.rawValue
+        }
+
+        init?(hour: Int) {
+            for choice in MajorMeal.allCases {
+                if choice.hours.contains(hour) {
+                    self = choice
+                    return
+                }
+            }
+
+            return nil
+        }
+
+        init(from date: Date) {
+            var calendar = Calendar.current
+            calendar.timeZone = TimeZone(identifier: "America/New_York")!
```

Comment:
Just in case the identifier changes or something:

```suggestion
            if let timeZone = TimeZone(identifier: "America/New_York") {
                calendar.timeZone = timeZone
            }
```